test/unit/alphabet/alphabet_test_template.hpp

```diff
@@ -194,33 +234,7 @@ TYPED_TEST_P(alphabet, comparison_operators)
     }
 }
 
-TYPED_TEST_P(alphabet, concept_check)
-{
-    EXPECT_TRUE(Semialphabet<TypeParam>);
-    EXPECT_TRUE(Semialphabet<TypeParam &>);
-
-    EXPECT_TRUE(WritableSemialphabet<TypeParam>);
-    EXPECT_TRUE(WritableSemialphabet<TypeParam &>);
-
-    EXPECT_TRUE(Semialphabet<TypeParam const>);
-    EXPECT_TRUE(Semialphabet<TypeParam const &>);
-
-    EXPECT_FALSE(WritableSemialphabet<TypeParam const>);
-    EXPECT_FALSE(WritableSemialphabet<TypeParam const &>);
-
-    EXPECT_TRUE(Alphabet<TypeParam>);
-    EXPECT_TRUE(Alphabet<TypeParam &>);
-
-    EXPECT_TRUE(WritableAlphabet<TypeParam>);
-    EXPECT_TRUE(WritableAlphabet<TypeParam &>);
-
-    EXPECT_TRUE(Alphabet<TypeParam const>);
-    EXPECT_TRUE(Alphabet<TypeParam const &>);
-
-    EXPECT_FALSE(WritableAlphabet<TypeParam const>);
-    EXPECT_FALSE(WritableAlphabet<TypeParam const &>);
-}
-
-REGISTER_TYPED_TEST_CASE_P(alphabet, alphabet_size_, default_value_constructor, global_assign_rank, global_to_rank,
-    copy_constructor, move_constructor, copy_assignment, move_assignment, swap, global_assign_char, global_to_char,
-    global_char_is_valid_for, global_assign_char_strict, comparison_operators, concept_check);
+REGISTER_TYPED_TEST_CASE_P(alphabet, concept_check, type_properties, alphabet_size_, default_value_constructor,
```

Comment:
just a wish not a review blocker. if each registered test is in one line, the diff would easier to spot next time :D